### PR TITLE
Fixes for ref dbs mounts for docker in ips and unifire. Mem limit to emapper --dbmem flag

### PR DESCRIPTION
**Pull request by mberacochea**
I've also added a local profile with max_cpus and memory suitable for a laptop/workstation

### CODE REVIEW

File: modules/local/eggnog.nf [hunk 1]
```diff
@@ -43,8 +51,7 @@ process EGGNOG_MAPPER {
         --no_file_comments \
         --cpu ${task.cpus} \
         --tax_scope 'prokaryota_broad' \
-        --dbmem \
```
Comment (tgurbich) [inline modules/local/eggnog.nf:46]: This flag makes by far the biggest difference in the annotation step. It shouldn't be removed.

File: nextflow.config [hunk 2]
```diff
@@ -175,6 +175,14 @@ profiles {
     }
     test      { includeConfig 'conf/test.config'      }
 
+    local {
+        params {
+            // Any modern laptop / desktop should have at least...
+            max_memory = "12GB"
```
Comment (tgurbich) [inline nextflow.config:181]: Is it definitely possible to run the pipeline with 12Gb? Have you tested this?

Comment (mberacochea) [reply nextflow.config:181]: I was able to run the pipeline in my laptop with these settings.

Comment (tgurbich) [reply nextflow.config:181]: That's great!

**Review by tgurbich — APPROVED** [review]
Thanks a lot, Martin! Could you please merge it into `dev` first instead of `main`?

**Comment by mberacochea** [comment]
> Thanks a lot, Martin! Could you please merge it into `dev` first instead of `main`?

Yeah, I selected the wrong branch when I created the PR.

**Comment by mberacochea** [comment]
BTW, this PR should close #38 and #40

**Pull request merged**